packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.java

```diff
@@ -25,7 +25,7 @@
 import expo.modules.updates.manifest.ManifestMetadata;
 
 // this unused import must stay because of versioning
-import expo.modules.updates.UpdatesConfiguration;
```

Comment:
this import needs to stay, per the comment

Reply:
Oh man, android studio seemed to have removed this automatically...

Reply:
yeah, it does that whenever you do a refactor that includes this file 😞